v3/src/components/graph/hooks/graph-hooks.ts

```diff
@@ -50,29 +51,47 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
     yNumeric = yAxisModel as INumericAxisModel,
     refreshPointsRef = useCurrent(refreshPointPositions)
 
+  /* This routine is frequently called many times in a row when something about the graph changes that requires
+  * refreshing the plot's point positions. That, by itself, would be a reason to ensure that
+  * the actual refreshPointPositions function is only called once. But another, even more important reason is
+  * that there is no guarantee that when callRefreshPointPositions is invoked, the d3 points in the plot
+  * have been synched with the data configuration's notion of which cases are plottable. Delaying the actual
+  * plotting of points until the next event cycle ensures that the data configuration's filter process will
+  * have had a chance to take place. */
+  const timer = useRef<any>()
+  const callRefreshPointPositions = useCallback((selectedOnly:boolean) => {
+    if( timer.current) {
+      return
+    }
+    timer.current = setTimeout(() => {
+      refreshPointsRef.current(selectedOnly)
```

Comment:
We'll need to make sure that this call doesn't happen after the ref has been destroyed. I think the way to do that is to add a `useEffect` to handle cleanup:
```typescript
    useEffect(() => {
      return () => if (timer.current) clearTimeout(timer.current)
    }, [])

Reply:
Done.